crossbar/_logging.py

```diff
@@ -150,6 +153,70 @@ def StandardErrorObserver(event):
     return StandardErrorObserver
 
 
+def makeJSONObserver(outFile):
+    """
+    Make an observer which writes JSON to C{outfile}.
+    """
+    def _make_json(event):
+
+        return json.dumps({"text": formatEvent(event).replace(u"{", u"{{").replace(u"}", u"}}"),
+                           "level": event.get("log_level", LogLevel.info).name})
+
+    recordSeparator = u"\x1e"
+    return FileLogObserver(
+        outFile,
+        lambda event: u"{0}{1}".format(_make_json(event), recordSeparator)
+    )
+
+
+def makeLegacyDailyLogFileObserver(path, logoutputlevel):
+    """
```

Comment:
everything else in crossbar uses under_score style names; maybe more consistent to do so with this _logging module as well?